runusb

```diff
@@ -202,6 +211,10 @@ class RobotUSBHandler(USBHandler):
     def _watch_process(self) -> None:
         # Wait for the process to complete
         self.process.wait()
+        if self.process.returncode != 0:
+            self.logger.warning(f"Process exited with code {self.process.returncode}")
+        else:
+            self.logger.debug("Your code finished successfully.")
```

Comment:
Issue: This isn't debug information

```suggestion
            self.logger.info("Your code finished successfully.")
```

Reply:
8cf74e86736631c47726896a07e570793d0134a5